spec/index.html

```diff
@@ -283,9 +283,157 @@ <h2>Introduction</h2>
   <section id="basic-concepts" class="informative">
     <h2>Basic Concepts</h2>
 
-    <p>FIXME.</p>
+    <p>
+      To ease writing and collaborating on JSON-LD documents, it is a common practice
+      to serialize them as YAML.
+      This requires not only to register a media type to enable content negotiation
+      of linked data documents in YAML, but even to define the expected behavior of
+      applications that processes these documents, including fragment identifiers and
+      interoperability consideration.
+
+      This is because YAML is more flexible than JSON:
+
+      * it supports different encodings, including UTF-8, UTF-16 and UTF-32;
+      * it supports more data types than JSON;
+      * the structure of its documents, that is named YAML representation graph,
+        is a tree that can have cycles.
+    </p>
+
+    <p>
+      The first goal of this specification is to allow to process a JSON-LD document
+      and serialize it in YAML, and then back to JSON-LD without losing any information.
+
+      This is always possible, because a YAML representation graph can always represent
+      a non-cyclic tree, because JSON data types are a subset of YAML's, and because
+      JSON encoding is UTF-8.
+    </p>
+
+    <p data-format="markdown>Example: the JSON-LD document below
+
+    ```
+    {
+      "@context": "http://example.org/context.jsonld",
+      "@graph": [
+        {"@id": "http://example.org/1", "title": "Example 1"},
+        {"@id": "http://example.org/2", "title": "Example 2"},
+        {"@id": "http://example.org/3", "title": "Example 3"}
+      ]
+    }
+    ```
+
+    can be serialized as YAML as follows. Note that - since `@`
+    is a reserved character in YAML, entries containing it 
+    needs to be enclosed in double quotes.
```

Comment:
```suggestion
    can be serialized as YAML as follows. Note that entries 
    containing `@` MUST be enclosed in double quotes because
    `@` is a reserved character in YAML.
```

Reply:
@TallTed refactoring is ok, but we don't need the MUST at line 329 because we are just explaining how YAML works, not mandating something new.

Is  there an alternative to `needs to be enclosed`? Can we retain it?

Reply:
```suggestion
    can be serialized as YAML as follows.
    Note that entries 
    starting with  `@` need to be enclosed in double quotes or escaped because
    `@` is a reserved character in YAML.
```